3MI0800270/2023-03-07/zad2.cpp

```diff
@@ -0,0 +1,19 @@
+#include <fstream>
+#include <iostream>
+using namespace std;
+
+int difChars(const char filename[]) {
+    ifstream file(filename);
+    int counter = 0;
+    while (!file.eof()) {
+        char ch = file.get();
```

Comment:
If `get` fails to read a char (because there're no left) the return value is the special `EOF` (see: [here](https://cplusplus.com/reference/istream/istream/get/#return)). You should add handling for that case too :))